### ISSUE TITLE
Do a one-click deploy to Vercel

### ISSUE THREAD

**Issue by Enoch2090** [goniszewski/grimoire#9]
[Working with the Deploy Button](https://vercel.com/docs/deployments/deploy-button#)

**Comment by jspn-creative** [comment]
Pocketbase needs to run persistently on a VPS, but Vercel is primarily a static site host. For a Vercel deployment to work without extra configuration, it would have to rely on services like Vercel Postgres or another cloud-hosted database provider.

I like the idea of easily spinning it up to play with it, but if grimoire takes off then it's in the project creator's best interest to continue catering to the self-hosted crowd until they decide to turn it into a service or otherwise monetize it. One-click deployments or adapters would be a ton of work and obfuscate the path to creating a service-based platform around grimoire.

tl;dr One-click deployments are not possible for this project without considerable work (or vendor lock-in), and even then are probably not in the project creator's best interest.
